docs/api.rst

```diff
@@ -3290,6 +3302,12 @@ of :class:`enum.Enum`.
 
         The rule will timeout a user.
 
+    .. attribute:: block_member_interactions
+
+        Similar to :attr:`timeout` the rule will timeout a user except of having a duration.
```

Comment:
The wording here needs to be updated. It's very unclear what you're trying to say.

Reply:
What about this?
```
Similar to :attr:`timeout` the rule will timeout a user except of having a limit.
```

Reply:
I think the second is a good improvement. It might be good to reorder the sentence a bit for clarity:
```rst
Similar to :attr:`timeout`, except the user will be timed out indefinitely.
```
This also leads nicely into the next sentence, which is a bonus!